fake/fake_clock.go

```diff
@@ -252,7 +254,7 @@ func (f *Clock) removeWaiter(ch chan struct{}, abort bool) {
 
 // SleepUntil blocks until either ctx expires or until arrives.
 // Return value is false if context-cancellation/expiry prompted an
-// early return
+// early return.
```

Comment:
Hmm, I don't have a problem with adding a period here, but we should be consistent in the other doc-comments. (It appears that the others that were edited are currently missing periods)

Reply:
Sure -- I've gone though and fixed all the docblocks to include periods at the end (I'd say most of them did to begin with).